tests/bin/draft/dune

```diff
@@ -0,0 +1,3 @@
+(mdx
```

Comment:
Could use cram tests for new tests?

Reply:
If I remember correctly if we enable the new cram feature we have to upgrade all existing tests as we can't say for which .t file it should be enabled or disabled.

Reply:
Ah ok, make sense, let's stick to mdx then!